handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: drush clear cache
```

Comment:
@benjifisher - Didn't notice your previous commit. No worries

> Maybe there should be a handler to clear the site cache.

That's definitely out of the purview of this role; that would be something better handled by a role like `geerlingguy.drupal` if it were to depend on this role.